.github/workflows/release-go-crosscompile-task.yml

```diff
@@ -9,8 +9,8 @@ env:
   # The project's folder on Arduino's download server for uploading builds
   AWS_PLUGIN_TARGET: /arduino-fwuploader/
   ARTIFACT_NAME: dist
-  # See: https://github.com/actions/setup-go/tree/v2#readme
-  GO_VERSION: ^1.16.2
+  # See: https://github.com/actions/setup-go/tree/main#supported-version-syntax
+  GO_VERSION: "1.17"
```

Comment:
I think this change should be made universally throughout the project infrastructure:

https://github.com/arduino/arduino-fwuploader/blob/96b4a40085655bd653de94d3a18e7910000ad64e/.github/workflows/check-go-cross-build-task.yml#L5

https://github.com/arduino/arduino-fwuploader/blob/96b4a40085655bd653de94d3a18e7910000ad64e/.github/workflows/check-go-dependencies-task.yml#L6

https://github.com/arduino/arduino-fwuploader/blob/96b4a40085655bd653de94d3a18e7910000ad64e/.github/workflows/check-mkdocs-task.yml#L6

https://github.com/arduino/arduino-fwuploader/blob/96b4a40085655bd653de94d3a18e7910000ad64e/.github/workflows/test-go-integration-task.yml#L6

https://github.com/arduino/arduino-fwuploader/blob/96b4a40085655bd653de94d3a18e7910000ad64e/.github/workflows/test-go-task.yml#L6

https://github.com/arduino/arduino-fwuploader/blob/96b4a40085655bd653de94d3a18e7910000ad64e/go.mod#L3

Reply:
should we do it in another PR?

Reply:
I think that is a good idea, since it is a significant change not directly related to the purpose of this PR. If you do that, I would recommend leaving this workflow and `.github/workflows/publish-go-tester-task.yml` at their previous Go ^1.16.2 version since I don't think it is beneficial to introduce even greater inconsistency in the versions used throughout the project's infrastructure.